File: backend/java/jx/commons/sochina-commons-base/src/main/java/com/sochina/base/domain/properties/BaseProperties.java
package com.sochina.base.domain.properties;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class BaseProperties implements Serializable {
    private static final long serialVersionUID = 1L;
    private boolean enable = true;
    private int order = -1;
    private List<String> excludeUrl = new ArrayList<>();
}

File: backend/kotlin/second/commons/base/src/main/kotlin/com/sochina/base/properties/BaseProperties.kt
package com.sochina.base.properties

import java.io.Serializable

/**
 * @author sochina-heart
 */
open class BaseProperties: Serializable {

    var enable: Boolean = true

    var excludeUrl: MutableList<String>? = null;

    var order: Int = -1
}